packages/iris-grid/src/mousehandlers/IrisGridCopyCellMouseHandler.ts

```diff
@@ -0,0 +1,60 @@
+import {
+  Grid,
+  GridMouseHandler,
+  GridPoint,
+  EventHandlerResult,
+  GridMouseEvent,
+} from '@deephaven/grid';
+import { ContextActionUtils } from '@deephaven/components';
+import IrisGrid from '../IrisGrid';
+
+class IrisGridCopyCellMouseHandler extends GridMouseHandler {
+  private irisGrid: IrisGrid;
+
+  constructor(irisGrid: IrisGrid) {
+    super();
+
+    this.irisGrid = irisGrid;
+    this.cursor = null;
+  }
+
+  onClick(
+    gridPoint: GridPoint,
+    grid: Grid,
+    event: GridMouseEvent
+  ): EventHandlerResult {
+    if (
+      event.altKey &&
+      !ContextActionUtils.isModifierKeyDown(event) &&
+      !event.shiftKey
+    ) {
+      this.cursor = null;
+      if (gridPoint.columnHeaderDepth !== undefined) {
+        this.irisGrid.copyColumnHeader(gridPoint.column);
```

Comment:
We should be passing the `columnHeaderDepth` to the `copyColumnHeader` here, and copying the one at the depth you've selected. Right now with a column group created it always copies to lowest header.
Group columns under "Organize Columns" in the menu to try it out.